brigade/core/__init__.py

```diff
@@ -123,8 +123,8 @@ def run(self, task, num_workers=None, **kwargs):
             **kwargs: additional argument to pass to ``task`` when calling it
 
         Raises:
-            :obj:`brigade.core.exceptions.BrigadeExceptionError`: if at least a task fails
-              and self.config.raise_on_error is set to ``True``
+            :obj:`brigade.core.exceptions.BrigadeExecutionError`: if at least a task fails
+              and self.raise_on_error is set to ``True``
```

Comment:
You replaced `self.config.raise_on_error` here with `self.raise_on_error`. Was that a mistake?